exercises/simple-linked-list/introduction.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
+
+You are a software developer at a music streaming company, and you've been tasked with creating a playlist feature for your music player application.
+
+You decide to make a simple prototype that uses a singly-linked list to create a playlist that users can listen to as is, or in reverse order.
+
+For now you're just working on getting the basic functionality right, so this version won't work with realistic data.
```

Comment:
I don't get this sentence. What is it about the exercise that means we need it?

Reply:
Because the previous instructions say to use an array of numbers, so all the existing implementations will probably have numbers, not song data, as the data being worked on in the exercise.

Reply:
Gotcha. How about we say "Each of the songs is identified by a unique number" instead?

Reply:
Yeah, that works, I think. I'll take another stab at this tomorrow, and then ping you (again) once I think I have something that is usable.